hoot-core/src/main/cpp/hoot/core/algorithms/WayMatchStringMerger.cpp

```diff
@@ -245,7 +245,7 @@ void WayMatchStringMerger::mergeIntersection(ElementId scrapNodeId)
   _moveNode(scrapNodeId, wl1);
 
   // notify _sublineMapping that way string 1 changed.
-  _rebuildWayString1();
+  _rebuildWayString(WayNumber::Way1);
```

Comment:
This is typical of the changes I made, created a function that can be called with a different `WayNumber` parameter to deal with WayString1 or WayString2.